src/utilities/dockerUtils.ts

```diff
@@ -26,14 +26,17 @@ export async function getDeploymentConfig(): Promise<DeploymentConfig> {
   const imageNames = core.getInput("images").split("\n") || [];
   const imageDockerfilePathMap: { [id: string]: string } = {};
 
-  //Fetching from image label if available
-  for (const image of imageNames) {
-    try {
-      imageDockerfilePathMap[image] = await getDockerfilePath(image);
-    } catch (ex) {
-      core.warning(
-        `Failed to get dockerfile path for image ${image.toString()}: ${ex} `
-      );
+  const pullImages = core.getInput("pull-images").toLowerCase() === "true";
```

Comment:
Actually the default is how you'd like it because of https://github.com/roehrijn/k8s-deploy/blob/feature/image-pull-switchable/action.yml#L21.
But to make it more explicit, I can change that according to your proposal, of course!

Reply:
Kind of. For example, if a user passes in "correct" instead of true then it would default to false in this case. With our updated code it will default correctly (even if the user doesn't have a correct input).

Reply:
Ok, understood and changed.